src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const element = document.createElement('div');
+  const body = document.querySelector('body');
+  const p = document.createElement('p');
+  const h2 = document.createElement('h2');
+
+  element.classList.add('notification', type);
+  h2.classList.add('title');
+
+  body.prepend(element);
```

Comment:
Consider appending the `element` to the `body` after setting its styles and content. This can help avoid unnecessary reflows and repaints in the browser.